src/Watchable.cpp

```diff
@@ -41,10 +41,14 @@ Watchable* Movie::getNextWatchable(Session& s) const
   return nullptr;
 }
 
-std::string Movie::toString() const
+std::string Movie::toStringWOTags() const
 {
-    // According to the "Inglorious Basterds 153 minutes [War, Western]" format
-    return name+" "+std::to_string(getLength())+" minutes "+getTagsString();
+    // According to the "Inglorious Basterds 153 minutes" format
+    return name+" "+std::to_string(getLength())+" minutes ";
+}
+
+std::string Movie::toString() const {
+    return toStringWOTags() + getTagsString();
```

Comment:
nice!!

Reply:
but you shouldn't print time as well...
so call it toStringName, and then toString will be
 `toStringName() + " "+std::to_string(getLength())+" minutes "+getTagsString()`